### DISCUSSION TITLE
🎉 GPT Discord is now featured on GitHub Education's Community Exchange 🎉

### DISCUSSION THREAD

**Discussion by ericdrosado** [Kav-K/GPTDiscord#415] [General]
👋 Hi @Kav-K ! I just wanted to let you know that **GPTDiscord** is now featured on GitHub Education's [Community Exchange](https://education.github.com/globalcampus/exchange). 🎉 

Congratulations and thank you for being an open-source Maintainer!

**Comment by Kav-K** [comment]
Hi @ericdrosado, I appreciate it! Thank you!!
